test/loopback-dl.test.js

```diff
@@ -456,6 +456,15 @@ describe('DataSource define model', function () {
     done(null, User);
   });
 
+  it('should report validation error for unknown properties if strict=validate', function() {
```

Comment:
Shouldn't this part really be a describe?

> if strict=validate

...something like

```js
describe('strict validation mode', ...
```

...or similar.